Source/CustomVoice.cpp

```diff
@@ -87,13 +110,21 @@ void CustomVoice::prepareToPlay (double sampleRate, int samplesPerBlock, int num
     setGain (-25.0);
 }
 
-// set ADSR envelope
+// Sets the attack, decay, sustain, release values of the volume envelope.
+//
+// @param params: A set of attack, decay sustain, release values in an ADSR::Parameters
+// object for the volume envelope to be set to.
 void CustomVoice::setADSR (juce::ADSR::Parameters parameters)
 {
     envelope.reset();
     envelope.setParameters (parameters);
 }
 
+// Changes which active oscillator between sine, square, saw, and triangle the
+// voice is using.
```

Comment:
I would rephrase this to something like: "Changes the active oscillator the voice is using between sine, square, saw, and triangle.